cuML/src/pca/pca.h

```diff
@@ -128,23 +110,18 @@ void pcaFit(math_t *input, math_t *components, math_t *explained_var,
  * @output param mu: mean of all the features (all the columns in the data). Size n_cols * 1.
  * @output param noise_vars: variance of the noise. Size 1 * 1 (scalar).
  * @input param prms: data structure that includes all the parameters from input size to algorithm.
- * @input param cublas_handle: cublas handle
- * @input param cusolver_handle: cusolver handle
  */
 template<typename math_t>
-void pcaFitTransform(math_t *input, math_t *trans_input, math_t *components,
-		math_t *explained_var, math_t *explained_var_ratio,
-		math_t *singular_vals, math_t *mu, math_t *noise_vars, paramsPCA prms,
-		cublasHandle_t cublas_handle, cusolverDnHandle_t cusolver_handle) {
-
-	pcaFit(input, components, explained_var, explained_var_ratio, singular_vals,
-			mu, noise_vars, prms, cublas_handle, cusolver_handle);
-
-	pcaTransform(input, components, trans_input, singular_vals, mu, prms,
-			cublas_handle);
-
-	signFlip(trans_input, prms.n_rows, prms.n_components, components,
-			prms.n_cols);
+void pcaFitTransform(const cumlHandle_impl& handle, math_t *input,
+                     math_t *trans_input, math_t *components,
+                     math_t *explained_var, math_t *explained_var_ratio,
+                     math_t *singular_vals, math_t *mu, math_t *noise_vars,
+                     paramsPCA prms) {
+    pcaFit(handle, input, components, explained_var, explained_var_ratio, singular_vals,
+           mu, noise_vars, prms);
+    pcaTransform(handle, input, components, trans_input, singular_vals, mu, prms);
+    signFlip(trans_input, prms.n_rows, prms.n_components, components,
```

Comment:
No stream or handle passed in. I see more instances of this below but will not call them all out.